### PR TITLE
Invalid password scenario added

### PR DESCRIPTION
**Pull request by rsita162010**
Invalid password scenario added

### CODE REVIEW

File: src/test/java/stepDefinitions/LoginPageStepDefintionSita.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package stepDefinitions;
+
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+
+public class LoginPageStepDefintionSita {
```
Comment (ShivamG1649) [inline src/test/java/stepDefinitions/LoginPageStepDefintionSita.java]: Why we have to create two step definition files ?
we can user one stepDefinition for one feature file 

for example if we have login.feature file , then we should have one step definition for this one

**Review by ShivamG1649 — COMMENTED** [review]
We should create only one step definition file for one feature

**Comment by rsita162010** [comment]
Added extra class in step definition, that is not required

**Comment by rsita162010** [comment]
Added extra class in step definition, that is not required

**Pull request closed without merging**